src/clj/game/core/prompts.clj

```diff
@@ -43,13 +44,13 @@
                (:card-title choices)
                (#{:credit :counter} choices)
                (pos? (count choices)))
-       (when waiting-prompt
+       (when (true? waiting-prompt)
          (add-to-prompt-queue
            state (if (= :corp side) :runner :corp)
            {:eid (select-keys eid [:eid])
             :card card
             :prompt-type :waiting
-            :msg (str "Waiting for " waiting-prompt)}))
+            :msg (str "Waiting for " (str (other-side side)) " to make a decision")}))
```

Comment:
Likewise, here you can say `(str "Waiting for " (if (true? waiting-prompt) (str (other-side side)) " to make a decision" waiting-prompt))` or something.

Reply:
We could also do something more complex here, if you think it would be helpful. You previously differentiated between "make a decision" and "choose an option" so we could have `(cond (and (true? waiting-prompt) (:choices ability)) " to choose an option" (true? waiting-prompt) " to make a decision" :else waiting-prompt)` or similar.

What do you think? Do we need that level of granularity?

Reply:
I think it would become too hidden in the internal implementation and not very clear to developers when to use which and why.

Reply:
Cool, good call.